http/analysis-service/config.py

```diff
@@ -4,6 +4,15 @@
 # AWS EC2 configuration
 AWS_REGION         = 'us-west-2'
 INSTANCE_TYPE      = 'c3.4xlarge'
+
+# EMR configuration
+# Master and slave instance types should be the same as the telemetry
+# setup bootstrap action depends on it to autotune the cluster.
+MASTER_INSTANCE_TYPE = 'c3.4xlarge'
+SLAVE_INSTANCE_TYPE = 'c3.4xlarge'
```

Comment:
Maybe just do `MASTER_INSTANCE_TYPE = INSTANCE_TYPE` if they must be the same. Or just use the `INSTANCE_TYPE` setting directly when launching the cluster.